src/validation.cpp

```diff
@@ -1766,8 +1794,9 @@ static bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockInd
             }
             nFees += txfee;
             if (!MoneyRange(nFees)) {
-                return state.DoS(100, error("%s: accumulated fee in the block out of range.", __func__),
-                                 REJECT_INVALID, "bad-txns-accumulated-fee-outofrange");
+                error("%s: accumulated fee in the block out of range.", __func__);
+                state.BadTx("bad-txns-accumulated-fee-outofrange");
+                return false;
```

Comment:
Shouldn't this be BadBlock? I don't see how'd you reach it without hitting a BadTx first though...